### PR TITLE
Update sdk_endpoints.md

### PR DESCRIPTION
**Pull request by davidbielik**
web sdk size update

### CODE REVIEW

**Review by wesleyorbin — DISMISSED** [review]
Should we add that the core is 35KB?

File: _docs/_user_guide/administrative/access_braze/sdk_endpoints.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ To configure the Braze Web SDK to use the appropriate endpoint for your integrat
 |---|---|
 | Android | 800 KB |
 | iOS | (IPA - Addition to App File) 1MB - 2MB; (Framework) 30MB |
-| Web | 35 KB |
+| Web | 50 KB |
```
Comment (davidbielik) [inline _docs/_user_guide/administrative/access_braze/sdk_endpoints.md:37]:
```suggestion
| Web | 35kb (core), 50kb (core + UI) |
```

File: _docs/_user_guide/administrative/access_braze/sdk_endpoints.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ To configure the Braze Web SDK to use the appropriate endpoint for your integrat
 |---|---|
 | Android | 800 KB |
 | iOS | (IPA - Addition to App File) 1MB - 2MB; (Framework) 30MB |
-| Web | 35 KB |
+| Web | 35kb (core), 50kb (core + UI) |
```
Comment (davidbielik) [inline _docs/_user_guide/administrative/access_braze/sdk_endpoints.md:37]:
```suggestion
| Web | 36kb (core), 50kb (core + UI) |
```

File: _docs/_user_guide/administrative/access_braze/sdk_endpoints.md (hunk 1 above)
Comment (wesleyorbin) [reply _docs/_user_guide/administrative/access_braze/sdk_endpoints.md]: Nitpick but `kb` should be `KB`. `b` means bits and `B` means bytes

File: _docs/_user_guide/administrative/access_braze/sdk_endpoints.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ To configure the Braze Web SDK to use the appropriate endpoint for your integrat
 |---|---|
 | Android | 800 KB |
 | iOS | (IPA - Addition to App File) 1MB - 2MB; (Framework) 30MB |
-| Web | 35 KB |
+| Web | 36kb (core), 50kb (core + UI) |
```
Comment (davidbielik) [inline _docs/_user_guide/administrative/access_braze/sdk_endpoints.md:37]:
```suggestion
| Web | 36KB (core), 50KB (core + UI) |
```

**Review by KellieHawks — APPROVED** [review]
lgtm!

**Pull request merged**